release-notes.txt

```diff
@@ -1,3 +1,366 @@
+    + c25519: add support (embedded x25519 crypto library)
+
+Boards
+------
+    + Arduino nano: added support
+    + nrf52832-mdk: added support
+    + PhyTec reel board: added support
+    + MicroPython pyboard v1.1: added support
+    + Hamilton: added support
+    + saml10-xpro and saml11-xpro: added riotboot support
+    + nrf52: added riotboot support
+    + ESP32 and stm32: added command to allow runtime configuration of uart mode
+    + ESP32 and ESP8266: add support for ESP-NOW network device driver
+    * ESP8266: freed up RAM by moving most .rodata sections from DRAM to IROM
+    + Arduino: added test to verify pin mappings
+    + b-l072z-lrwan1: enabled RTT feature
+    + STM32l0: added support for dual bank flashing with OpenOCD
+    * Autonomo: changed flasher to BOSSA
+
+CPU
+---
+    + stm32l0: added riotboot support
+    + stm32f0: added support for periph_pm
+    + stm32f3 and stm32l1: added stop and standby low power modes
+    + samd21: enabled idle modes
+    + sam0: added support for RWWEE on-chip memory
+    + Kinetis: added support for EA series and S9KEAZ129 MCU
+    + TI CC2650: added I2C periph implementation
+
+Device Drivers
+--------------
+    + i2c: added slave scan shell command
+    + USB: added peripheral driver for sam0_common
+    + USB: added device driver initialization function and API definition
+    + nrf5x temperature sensor: added driver
+    + nrf52: added initial 802.15.4 support
+    + add7746 capacitance sensor: added driver
+    + ds323x extremely accurate RTC: added driver
+    + ltc4150 coulomb counter: added driver
+    + sht2x temperature and humidity sensor: added driver
+    + sds011 dust sensor: added driver
+    + added high level driver for H bridge ICs to drive DC motors
+    * lpsxxx pressure and temperature sensor: add support for lps22hb and
+      lps25hb
+
+Build System / Tooling
+----------------------
+    + nrf52: added support for openocd
+    + added support for Texas Instruments UniFlash
+    * improved avrdude integration
+    * improved handling of flashing images at offset addresses
+    * allowed clean definition of file to be flashed
+    * improved Murdock's info-boards-supported failure message
+    + added test result caching to Murdock
+    + added support for socat as a RIOT terminal
+    * pyterm: specified use of python3 by default
+    + added script for backporting PRs during release testing
+
+Testing
+-------
+    + documentation: added initial guide to running tests
+
+Known issues
+============
+
+Networking related issues
+-------------------------
+#11295: RFC 3306 Unicast-Prefix-based-IPv6 Multicast with BorderRouter
+#11212: POSIX TCP API: bad file descriptor
+#11091: gnrc_minimal on nRF52DK do not work anymore
+#11033: 6lo: RIOT does not receive packets from Linux when short_addr is set
+#10969: net: netdev_driver_t::send() doc unclear
+#10927: gnrc_tftp: Missing minimum length check
+#10878: nrfmin can get stuck and never reach RX (while TX works)
```

Comment:
There is a fix for that https://github.com/RIOT-OS/RIOT/pull/11395, that should be merged today-ish

Reply:
Ok, #11395 did not fix all the issues that are mentioned in #10878 so we should leave it in.